code/modules/clothing/suits/labcoat.dm

```diff
@@ -18,12 +18,26 @@
 		/obj/item/storage/pill_bottle,
 		/obj/item/paper,
 
+		/obj/item/weapon/gun/,
 		/obj/item/storage/fancy/cigarettes,
 		/obj/item/tool/lighter,
 		/obj/item/weapon/baton,
 		/obj/item/restraint/handcuffs,
 		/obj/item/device/binoculars,
 		/obj/item/attachable/bayonet,
+		/obj/item/storage/belt/gun/m4a3,
+		/obj/item/storage/belt/gun/m44,
+		/obj/item/storage/belt/gun/mateba,
+		/obj/item/storage/belt/gun/smartpistol,
+		/obj/item/storage/backpack/general_belt,
+
+		/obj/item/device/flashlight,
+		/obj/item/device/healthanalyzer,
+		/obj/item/device/radio,
+		/obj/item/tank/emergency_oxygen,
+		/obj/item/tool/crowbar,
+		/obj/item/tool/pen,
+		/obj/item/storage/large_holster/machete,
```

Comment:
as well as all belt guns here and holsters

Reply:
They are actually, labcoats can't store anything. Unless i make it a subdivision of the marine jacket, it cant store the items literally every other suit slot item can